app/boutique-hotel-technikum/src/components/AboutPageCard.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <ion-col size="12" size-sm="12" size-md="6" size-lg="6" size-xl="6">
```

Comment:
From the Ionic documentation, you don't need to specify the size for bigger screens if it the same than a smaller one:

> Grid tiers are based on minimum widths, meaning they apply to their tier and all those larger than them (e.g., size-sm="4" applies to small, medium, large, and extra large devices).
https://ionicframework.com/docs/api/grid#overview